articles/custom-domains/additional-configuration.md

```diff
@@ -129,15 +149,22 @@ If you want to use SAML applications with your custom domain, you must update yo
 
 If you have an IdP-initiated authentication flow, you will need to update the URL used to invoke the IdP-initiated authentication flow to reflect the custom domain. Instead of `https://<TENANT>.auth0.com/saml/<CLIENTID>` you should use `https://<CNAME>/saml/<CLIENTID>`.
 
-## APIs
+If you use the Auth0 APIs, such as the Management API, the API identifier will use your default tenant domain name (such as `https://${account.namespace}/userinfo` and `https://${account.namespace}/api/v2/`)
 
-If you use Auth0 with a custom domain to issue Access Tokens for your APIs, then you must validate the JWT issuer(s) against your custom domains. For example, if you use the [express-jwt](https://github.com/auth0/express-jwt) middleware, you must do the following change.
+## Configure your WS-Fed Clients
 
-```js
-app.use(jwt({ 
-  issuer: 'https://YOUR-CUSTOM-DOMAIN',
-  //code omitted for brevity
-}));
-```
+If you want to use your WS-Fed applications with your custom domain with Auth0 as the IDP, you must update your Service Provider with new Identity Provider metadata from Auth0 (You can obtain the metadata reflecting the custom domain from: `https://<CUSTOM DOMAIN>/wsfed/FederationMetadata/2007-06/FederationMetadata.xml`).
```

Comment:
this doc https://auth0.com/docs/integrations/configure-wsfed-application should also have this info